wordcloud/wordcloud_cli.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+r"""Command-line tool to generate word clouds
+Usage::
+    $ cat word.txt | wordcloud_cli.py
+
+    $ wordcloud_cli.py --words=words.txt --stopwords=stopwords.txt
+"""
+import argparse
+import wordcloud as wc
+import numpy as np
+import sys
+import io
+import colorsys
+from PIL import Image
+from PIL import ImageColor
+
+def get_single_color_func(color):
```

Comment:
Something like ImageColorGenerator from color_from_image? It's giving different saturations for a fixed hue/luminance - SaturationColorGenerator in fixed_color.